bucket/srecord.json

```diff
@@ -0,0 +1,14 @@
+{
+    "homepage":  "http://srecord.sourceforge.net/",
+    "description": "The SRecord package is a collection of powerful tools for manipulating EPROM load files",
+    "version":  "1.64",
+    "license":  "GPL-3.0-only",
+    "url":  "https://sourceforge.net/projects/srecord/files/srecord-win32/1.64/srecord-1.64-win32.zip",
+    "hash":  "83ef7da6e114705b35d06edf4ce430414d14c0088a8fa6a273da8fe3282bc280",
+    "bin":  [
+        "srec_cat.exe",
+        "srec_cmp.exe",
+        "srec_info.exe"
+    ],
+    "checkver": "SRecord ([\\d.]+)"
```

Comment:
https://sourceforge.net/projects/srecord/rss?path=/ might be more reliable than the homepage. See https://github.com/ScoopInstaller/Main/blob/a2f604fae5a8ceddcdca50cca1452b156412aa7f/bucket/uncrustify.json#L17-L20 for an example.